sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobProperties.java

```diff
@@ -49,7 +49,7 @@
     private final Metadata metadata;
     private final Integer committedBlockCount;
 
-    BlobProperties(BlobGetPropertiesHeaders generatedHeaders) {
+    public BlobProperties(BlobGetPropertiesHeaders generatedHeaders) {
```

Comment:
Do you expect people to create this class? I'm curious why it is public. Missing documentation if you intend to keep it public.